.github/workflows/build.yaml

```diff
@@ -81,12 +81,20 @@ jobs:
     name: bazel test --build ${{ matrix.platform }}
     steps:
     - uses: actions/checkout@v2
+    # TODO(https://github.com/sputt/req-compile/issues/73): Fix buildbuddy
```

Comment:
Tried to limit this to CI but the secret seems to be borked since we still see issues in CI
- https://github.com/sputt/req-compile/issues/73